dev/com.ibm.ws.jpa_testframework/src/com/ibm/ws/testtooling/vehicle/web/AbstractDatabaseManagementServlet.java

```diff
@@ -162,115 +163,114 @@ private static void executeDDL(DataSource ds, UserTransaction tx, ClassLoader cl
 
         final PrintWriter pw = resp.getWriter();
         try {
+            // Set transaction timeout to 10 minutes
+            tx.setTransactionTimeout((int) Duration.ofMinutes(10).getSeconds());
             tx.begin();
-            final Connection conn = ds.getConnection();
-            final DatabaseMetaData dbMeta = conn.getMetaData();
-            dumpDBMeta(dbMeta);
 
-            final Statement stmt = conn.createStatement();
-            final String defaultSchemaName = (overrideDefaultSchema == null || "".equals(overrideDefaultSchema.trim())) ? //
-                            sanitize(dbMeta.getUserName()) : // Usually the default schema name
-                            sanitize(overrideDefaultSchema); // But always allow for test client to override
+            try (Connection conn = ds.getConnection()) {
+                final DatabaseMetaData dbMeta = conn.getMetaData();
+                dumpDBMeta(dbMeta);
```

Comment:
```suggestion
            try (Connection conn = ds.getConnection()) {
                final DatabaseMetaData dbMeta = conn.getMetaData();
                dumpDBMeta(dbMeta);
             }
```

suggestion: For a read-only function like getMetaData() I would suggest running this before the transaction. No reason this needs to be done during the transaction.